lib/challenge.js

```diff
@@ -48,7 +62,14 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet {
```

Comment:
You're missing the assignment operator (`=`) between `some_pet` and `{`. Are you experiencing any confusion with curly braces? They have several meanings depending on context. If you're unsure, come see me or someone else on the instructional team for clarification.